Utils/Dataflow/009_oracleConnector/README

```diff
@@ -23,3 +23,23 @@ A template for the configuration file can be obtained in Dataflow's
 corresponding directory:
 
   Utils/Dataflow/config/009.cfg.example
+
+3. Task Chain
+-------------
+The query prodsys2ES includes the subquery that obtains the task's chain_data:
```

Comment:
`...includes...` --> `...contains...`?
I am not sure which of them is better in this context, so just a suggestion.

Reply:
I can't find any solid reason to use any of the two options over another in this case, so I'll leave the text as is per my intuition.